### PR TITLE
Bug fix/gcb 232 wrong number of bubbles

### PR DESCRIPTION
**Pull request by Augustyniak**
I still want to write unit tests for fetching logic - I will do that in a separate PR as I don't want to block release of the bonusly widget.

### CODE REVIEW

File: GrandCentralBoard/Widgets/Bonus/WrappingRequestTemplate.swift [hunk 1]
```diff
@@ -16,7 +16,9 @@ class WrappingRequestTemplate<T: RequestTemplateProtocol> : RequestTemplateProto
     private let requestTemplate: T
 
     init(requestTemplate: T, queryParameters: parameters) {
-        self.queryParameters = queryParameters
+        var mQueryParameters = queryParameters
+        mQueryParameters.merge(queryParameters)
+        self.queryParameters = mQueryParameters
```
Comment (michallaskowski) [inline GrandCentralBoard/Widgets/Bonus/WrappingRequestTemplate.swift]: Hungarian notation :)?

Comment (Augustyniak) [inline GrandCentralBoard/Widgets/Bonus/WrappingRequestTemplate.swift]: m - mutable. I can expand this shortcut if you want :)

Comment (Augustyniak) [inline GrandCentralBoard/Widgets/Bonus/WrappingRequestTemplate.swift]: but you are right - it is somehow similar to Hungarian :)

Comment (michallaskowski) [inline GrandCentralBoard/Widgets/Bonus/WrappingRequestTemplate.swift]: I think you could just shadow the parameter:
```
var queryParameters = queryParameters
queryParameters.merge(queryParameters)
self.queryParameters = queryParameters
```

Comment (michallaskowski) [inline GrandCentralBoard/Widgets/Bonus/WrappingRequestTemplate.swift]: Or maybe better not, let's leave it.

**Pull request merged**